### ISSUE TITLE
Error messages get logged

### ISSUE THREAD

**Issue by mmattel** [owncloud/music#601]
Music tells me that there are new titles to be scanned. 
When pressing scan on the browser I get "thousands" of error messages logged with the same style:
```
13T10:35:24+01:00","remoteAddr":"u.v.w.x","user":"user_1","app":"PHP","method":"GET"
,"url":"\/apps\/music\/api\/album\/29878\/cover","message":"Error while sending QUERY packet. PID=38959 at 
\/var\/www\/owncloud\/lib\/composer\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Driver\/PDOStatement.php#91"}
```

**Comment by paulijar** [comment]
With a quick web search, it seems that MySQL can emit that kind of error at least when
a) the size of the database query is larger than `max_allowed_packet`
b) there are too many database requests at the same time.

The log message tells us also that this has happened when the client was trying to get the cover art of one album. In this context, the option a) doesn't seem likely because we cache the cover image to the database only if it's size is less than 100 KB, and `max_allowed_packet`  limit shouldn't be that low (in my installation it´s 16 MB). I guess that the option b) is possible if the database somehow gets overwhelmed when simultaneously new entries are inserted and the previously inserted cover images are fetched.

Does your music collection actually contain thousands of albums?

**Comment by mmattel** [comment]
yes, >12k titles...

> when simultaneously new entries are inserted and the previously inserted cover images are fetched

hmmm...

**Comment by paulijar** [comment]
Is this issue still present on the new release v0.5.6?

**Comment by paulijar** [comment]
I assume that this is another instance of the same problem as #719. This should have got fixed with Music v0.12.0.
